res/values/strings.xml

```diff
@@ -465,6 +465,7 @@
     <string name="confirmation_remove_files_alert">Do you really want to remove the selected items?</string>
     <string name="confirmation_remove_folders_alert">Do you really want to remove the selected items and their contents?</string>
     <string name="maintenance_mode">Server in maintenance mode</string>
+    <string name="failed_upload_quota_exceeded_text">Not enough free space</string>
```

Comment:
I've copied the message from iOS solution few days ago but I've just checked the code of a similar issue on iOS and has changed, now is: "Not enough free space in your account". This is a proper message. What do you think?